dev-scripts/env-for-minishift

```diff
@@ -7,3 +7,4 @@ export OPENSHIFT_NAMESPACE_URL=${OPENSHIFT_NAMESPACE_URL:-"${CHE_OPENSHIFT_PROJE
 export CHE_LOG_LEVEL=${CHE_LOG_LEVEL:-"DEBUG"}
 export CHE_DEBUGGING_ENABLED=${CHE_DEBUGGING_ENABLED:-"true"}
 export FABRIC8_ONLINE_PATH=${FABRIC8_ONLINE_PATH:-"${HOME}/github/fabric8-online/"}
+export CHE_KEYCLOAK_DISABLED=${CHE_KEYCLOAK_DISABLED:-"true"}
```

Comment:
@ibuziuk I requested to @davidfestal to do so. Let me try to explain why: I would like to make it easy for a user to run Che on minishift. And I'm not talking about a Red Hat developer but for anyone that want to try it out. It should be able to use it even if he has not registered on OSIO.